app/uk/gov/hmrc/bindingtariffclassification/repository/SearchMapper.scala

```diff
@@ -64,7 +64,17 @@ class SearchMapper @Inject()(appConfig: AppConfig) {
       filter.statuses.map(filteringByStatus)
     ).filter(_.isDefined).map(_.get)
 
-    JsObject(params)
+    val query: Map[String, JsValue] = params.groupBy(_._1) // Group by Key
```

Comment:
Looks good, do we need any extra SearchMapper Unit tests?

Reply:
There was a broken existing test - fixed now